src/Suggest/Phrase.php

```diff
@@ -140,7 +140,7 @@ public function setSmoothingModel(string $model, array $params): Phrase
      */
     public function addCandidateGenerator(AbstractCandidateGenerator $generator): Phrase
```

Comment:
What about deprecating this method, and instead add a new one `addDirectGenerator(DirectGenerator $generator)` as this suggester only allow configuring `direct_generator`. That would this way avoid to trick to `toArray` method.